enterprise/models.py

```diff
@@ -1267,13 +1268,23 @@ def link_pending_enterprise_user(self, user, is_user_created):
                     user=user,
                     enterprise_customer=enterprise_customer_user.enterprise_customer,
                 ))
+
+                enterprise_customer_user.active = True
+                enterprise_customer_user.save()
+                enterprise_customer_user.inactivate_other_customers(
```

Comment:
Could we push this down into a more common place, like `EnterpriseCustomerUser.save()` or the model manager or something?  That way future writers of code that set `active = True` don't have to worry about this?

Reply:
@iloveagent57 I actually debated this and kept going back and forth about it haha. I ultimately decided not to put it in `EnterpriseCustomerUser.save()` as I felt it might have been inappropriate to tightly couple inactivating other customers on every `EnterpriseCustomerUser` save. The primary concern I had was if ECS or engineering manually updates an `EnterpriseCustomerUser` record or it somehow otherwise gets updated with the expectation of no side effects.

Reply:
What if we encapsulated the whole thing in a function like `EnterpriseCustomerUser.set_active()`, which would set that record with `active = True` and then call `inactivate_other_customers()`.?

Reply:
@iloveagent57 Then again, I suppose we would only call it when `active` changes from false to true for a given `EnterpriseCustomerUser`. Which wouldn't be on every `EnterpriseCustomerUser.save()`. I'm going to explore this a little further and re-evaluate I think.

Reply:
@iloveagent57 That's also a possibility. I'll sleep on it and see how ambitious I am in the morning haha!